application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -769,14 +769,20 @@ type Leg {
   """
   For transit legs, intermediate stops between the Place where the leg
   originates and the Place where the leg ends. For non-transit legs, null.
-  Returns Place type, which has fields for e.g. departure and arrival times
+  Returns Place type, which has fields for e.g. departure and arrival times.
+    
+  The `include` parameter allows filtering of the returned places by stop type. If not provided, the
+  field returns all types. An empty list is not permitted.
   """
-  intermediatePlaces: [Place]
+  intermediatePlaces(include: [StopType!]): [Place]
```

Comment:
Perhaps it would make sense to either create own enum for this call since this returns other things than stops as well or give this a more specific name or wrap this in some include: { stopTypes: [StopType!] } structure. If we decide to have one big enum, exclude would feel more natural to use than an include in this case.